core/api/errors/validation_error.py

```diff
@@ -0,0 +1,25 @@
+from typing import Union
+
+from fastapi.exceptions import RequestValidationError
```

Comment:
could prettify the imports. That is sort ascending on length of line

Reply:
can you explain what you mean by this? i.e. add a suggestion of what you believe it should look like?

Reply:
import a
import aa
import aaa
import aaaa.aaaa

Looks prettier. I know vanity :D